aries-site/src/examples/templates/forms/SignUpExample.js

```diff
@@ -41,7 +40,7 @@ export const SignUpExample = () => {
           </Heading>
           <Text>for a Hewlett Packard Enterprise account</Text>
         </Header>
-        <Main
```

Comment:
was there a particular reason we switched out `Main` with `Box` ?

Reply:
Main should be reserved for the primary content container on a document. Per spec, a document should only have one Main.